src/libs/actions/Report.ts

```diff
@@ -3673,6 +3673,30 @@ function setGroupDraft(newGroupDraft: Partial<NewGroupChatDraft>) {
     Onyx.merge(ONYXKEYS.NEW_GROUP_CHAT_DRAFT, newGroupDraft);
 }
 
+function setTaskDataForNewAssingee(assigneeLogin: string, accountID: number | undefined = undefined) {
```

Comment:
```suggestion
function setNewOptimisticAssignee(assigneeLogin: string, assigneeAccountID: number) {
```

Reply:
Or `setOptimisticReportAndPersonalDetailsForNewAssignee`

Reply:
@s77rt  What do you think about `setOptimisticDataForNewAssignee`?

Reply:
Let's not use `data` because it does not tell us anything and we would have to read the function to understand what `data` means

Reply:
Thanks, I think `setNewOptimisticAssignee` is fine.

Reply:
@s77rt I updated.